CHANGELOG.md

```diff
@@ -1,16 +1,24 @@
 # web-ledger-client ChangeLog
 
-### 6.0.1 - 2021-xx-xx
+## 7.0.0 - 2022-xx-xx
 
-NOTE FOR NEXT RELEASE: 6.0.0 was erroneously released, next available version
-is 6.0.1
+### Changed
+- **BREAKING**: Convert to module (ESM).
+- **BREAKING**: Require Node.js >=14.
+- Update dependencies.
+- Lint module.
+
+## 6.0.0 - NONE
+
+### Notes
+- 6.0.0 was erroneously released, next available version is 6.0.1.
```

Comment:
This says 6.0.1 was released but the above wipes it out here ... can this be looked into?

Reply:
There was a mistaken 6.0.0 release that is marked as deprecated.  I was just going to avoid confusion and release the next version as 7.0.0.  I simplified the note here.